cuba.yml

```diff
@@ -405,3 +405,23 @@ CUBA_KEYS:
     definition: Work of adhesion per unit contact area
     shape: [1]
     type: double
+
+  MAJOR:
+    definition: Major attribute of an entity.
+    shape: [8]
+    type: string
+
+  MINOR:
+    definition: Minor version of a software tool with semantic versioning.
+    shape: [8]
+    type: string
+
+  PATCH:
+    definition: Patch version of a software tool with semantic versioning.
+    shape: [8]
+    type: string
+
+  FULL:
+    definition: Full version of a software tool with any versioning scheme.
```

Comment:
same as above, this should be a general definition of a FULL keyword, depending on the association with another entity, it gains a specific meanning. The specific version definition can be added to the definition in simphony_metada.yml